soda/core/soda/cli/cli.py

```diff
@@ -319,7 +319,13 @@ def update_dro(
     if filter is not None:
         filter_clause = f"WHERE {filter}"
 
+    sample = distribution_dict.get("sample")
+    sample_clause = sample if sample is not None else ""
+
     if dataset_name and column_name and distribution_type:
+        query = f"SELECT {column_name} FROM {dataset_name} {sample_clause} {filter_clause}"
```

Comment:
I guess the added whitespace between `dataset_name` and `filter_clause` doesn't mess up the sql right (if `sample_clause` is empty)?